packages/cli/test/integration/auth.api.test.ts

```diff
@@ -84,6 +86,28 @@ describe('POST /login', () => {
 		const authToken = utils.getAuthToken(response);
 		expect(authToken).toBeDefined();
 	});
+
+	test('should throw AuthError for non-owner if not within users limit quota', async () => {
+		jest.spyOn(Container.get(License), 'isWithinUsersLimitQuota').mockReturnValueOnce(false);
+		const member = await testDb.createUserShell(globalMemberRole);
+		try {
+			await authAgent(member).get('/login');
+		} catch (error) {
+			expect(error.httpStatusCode).toBe(401);
+		}
```

Comment:
Looking at other tests checking for errors, I think we care more about the error in the response rather than the error itself:

```ts
const response = await authAgent(member).get('/login');
expect(response.statusCode).toBe(401);
```